src/phylum/ci/lockfile.py

```diff
@@ -1,10 +1,16 @@
 """Define a lockfile implementation.
 
 A Phylum project can consist of multiple lockfiles.
-This class represents a single lockfile.
+This module/class represents a single lockfile.
+
+For historical purposes, this module uses "lockfile" language instead of the
+more general term "dependency file." Both lockfiles and manifests are supported
+and the language has been changed where it is externally visible (e.g., log and
+help output) but kept as "lockfile" internally (e.g., code/variable names).
```

Comment:
What's the reason for keeping variable names etc the same? I'd assume just too much work? For maintenance purposes it would probably be good to have correct variable names.

Reply:
Correct...it is _a lot_ to change. I would like to make the change and might still do so...but in a different PR so as not to clutter the "meat" of this one with all those changes. One other reason for keeping `lockfile` internally was the observation that the `phylum-dev/cli` repo also does. I plan to bring this up for discussion during a team meeting...to ensure we are all on the same page about naming things and consistency.

Reply:
Conclusion from the sync: Definitive naming isn't decided upon yet. So probably better to keep it the way it is for now.